### PR TITLE
Post slack message for s4s pipeline including with checks

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**

### CODE REVIEW

File: src/brain/gocdSlackFeeds/index.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { GoCDResponse } from '@/types';
 import { DeployFeed } from './deployFeed';
 
 const ENGINEERING_PIPELINE_FILTER = [
+  'deploy-getsentry-backend-s4s',
```
Comment (IanWoodard) [inline src/brain/gocdSlackFeeds/index.ts:15]: Should this be a constant?

**Review by IanWoodard — APPROVED** [review]

File: src/brain/gocdSlackFeeds/index.ts (hunk 1 above)
Comment (mattgauntseo-sentry) [reply src/brain/gocdSlackFeeds/index.ts:15]: No it doesn't add any value - the constants are largely there to make it is easy to run eng-pipes in a test environment (i.e. diff gocd, diff github org, diff repos)

File: src/brain/gocdSlackFeeds/index.ts [hunk 2]
```diff
@@ -62,11 +63,6 @@ const engineeringFeed = new DeployFeed({
       return false;
     }
 
-    // We don't care about the checks stage failing as it is flaky and noisy.
-    if (pipeline.stage.name.toLowerCase() === 'checks') {
-      return false;
-    }
-
```
[the remark this answers is not in the document]
Comment (mattgauntseo-sentry) [reply src/brain/gocdSlackFeeds/index.ts:69]: I agree, but it seems like no noise is also causing issues. Any other ideas?

[the remark this answers is not in the document]
Comment (IanWoodard) [reply src/brain/gocdSlackFeeds/index.ts:69]: We could adjust how we show different errors to demonstrate the varying severity.

File: src/brain/gocdSlackFeeds/index.ts (hunk 1 above)
Comment (IanWoodard) [reply src/brain/gocdSlackFeeds/index.ts:15]: fair enough

File: src/brain/gocdSlackFeeds/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mattgauntseo-sentry) [reply src/brain/gocdSlackFeeds/index.ts:69]: The severity should probably be tied the number of times the deploy has failed in which case we could not post a message until the deploy fails twice in a row, but at that point, being noisy may have the same or better affect and it's harder for people to ignore.

[the remark this answers is not in the document]
Comment (IanWoodard) [reply src/brain/gocdSlackFeeds/index.ts:69]: fair enough, I guess let's see how this does

**Pull request merged**